src/test/checkqueue_tests.cpp

```diff
@@ -149,7 +148,7 @@ static void Correct_Queue_range(std::vector<size_t> range)
 {
     auto small_queue = MakeUnique<Correct_Queue>(QUEUE_BATCH_SIZE);
     boost::thread_group tg;
-    for (auto x = 0; x < nScriptCheckThreads; ++x) {
```

Comment:
6e6cb418accf9286148cff9f1966fca27d38e0d7

Locally it builds if `#include <validation.h>` is removed.

Reply:
fixed